README.md

```diff
@@ -9,16 +9,9 @@ The MoveIt! Motion Planning Framework **for ROS 2.0**
 - [Get Involved](http://moveit.ros.org/documentation/contributing/)
 
 ## Milestones
-- [ ] Setup instructions
-  - [ ] Install instructions
-    - [ ] Ubuntu 18.04
-    - [ ] OS X 10.14
-- [ ] Update/setup infrastructure for development
-  - [x] Delete metapackages
-  - [x] Upgrade continuous integration for ROS 2.0
-  - [x] Refactor/cleanup folder hierarchy
+- [ ] Install instructions for Ubuntu 18.04 in README.md
+- [ ] Upgrade continuous integration for ROS 2.0
```

Comment:
Add item removed, not sure where though

```suggestion
- [ ] Upgrade continuous integration for ROS 2.0
- [x] Update/setup infrastructure for development
    - [x] Delete metapackages
    - [x] Upgrade continuous integration for ROS 2.0
    - [x] Refactor/cleanup folder hierarchy
```

Reply:
I expanded these with more details, below:
>   - [ ] Delete excesses packages that are left over from rosbuild stacks: moveit_runtime, moveit_plugins, moveit_ros
>  - [ ] Rename non-package folders: 
>    - [ ] rename moveit_planners to planners
>    - [ ] rename moveit_plugins to controller_interfaces
>  - [ ] Restructure folder layout of moveit repo:
>    - [ ] flatten moveit_ros folder to root of repo
>    - [ ] rename all moveit_ros folders with moveit_ros prefix

Reply:
That looks good @davetcoleman